src/index.js

```diff
@@ -134,7 +135,8 @@ class Griddle extends Component {
 
     this.store = createStore(
       reducers,
-      initialState
+      initialState,
+      applyMiddleware(...plugins.map(p => p.reduxMiddleware), ...reduxMiddleware) 
     );
```

Comment:
I don't remember… does spreading an array also flatten nested arrays? If it doesn't, this will give us arrays of middleware as arguments. Does that work?